src/planner/binder/statement/bind_insert.cpp

```diff
@@ -54,18 +54,21 @@ void ReplaceDefaultExpression(unique_ptr<ParsedExpression> &expr, const ColumnDe
 	expr = ExpandDefaultExpression(column);
 }
 
-void QualifyColumnReferences(unique_ptr<ParsedExpression> &expr, const string &table_name) {
+void QualifyColumnReferences(unique_ptr<ParsedExpression> &expr, const string &table_name, bool toggle = true) {
 	// To avoid ambiguity with 'excluded', we explicitly qualify all column references
-	if (expr->type == ExpressionType::COLUMN_REF) {
+	if (toggle && expr->type == ExpressionType::COLUMN_REF) {
 		auto &column_ref = expr->Cast<ColumnRefExpression>();
 		if (column_ref.IsQualified()) {
 			return;
 		}
 		auto column_name = column_ref.GetColumnName();
 		expr = make_uniq<ColumnRefExpression>(column_name, table_name);
 	}
+	if (expr->type == ExpressionType::LAMBDA) {
+		toggle = false;
```

Comment:
I don't quite understand the need for this `toggle` - we never toggle it back on. Can't we `return` here instead?